sdk/confidentialledger/azure-confidentialledger/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+
+## 0.0.1 (Unreleased)
```

Comment:
I believe the initial version should be `1.0.0b1` (make sure the version in `_version.py` is also updated)